modules/mobile/app/controllers/mobile/v0/community_care_providers_controller.rb

```diff
@@ -0,0 +1,100 @@
+# frozen_string_literal: true
+
+module Mobile
+  module V0
+    class CommunityCareProvidersController < ApplicationController
+      DEFAULT_PAGE_NUMBER = 1
+      DEFAULT_PAGE_SIZE = 10
+      RADIUS = 60 # default value used in web app
+      SPECIALTIES = {
+        primaryCare: %w[207QA0505X 363LP2300X 363LA2200X 261QP2300X],
+        foodAndNutrition: %w[133V00000X 133VN1201X 133N00000X 133NN1002X],
+        podiatry: %w[213E00000X 213EG0000X 213EP1101X 213ES0131X 213ES0103X],
+        optometry: %w[152W00000X 152WC0802X],
+        audiologyRoutineExam: %w[231H00000X 237600000X 261QH0700X],
+        audiologyHearingAidSupport: %w[231H00000X 237600000X]
+      }.with_indifferent_access.freeze
+
+      def index
+        ccps = ppms_api.facility_service_locator(locator_params)
+        community_care_providers = ccps.map do |record|
+          Mobile::V0::CommunityCareProvider.new(
+            id: record.id,
+            name: record.provider_name,
+            address: Address.new(
+              street: record.address_street,
+              city: record.address_city,
+              state: record.address_state_province,
+              zip_code: record.address_postal_code
+            ),
+            distance: record.miles
+          )
+        end
+        page_records, page_meta_data = paginate(community_care_providers)
+        serialized = Mobile::V0::CommunityCareProvidersSerializer.new(page_records, page_meta_data)
+        render json: serialized, status: :ok
+      end
+
+      private
+
+      def ppms_api
+        FacilitiesApi::V1::PPMS::Client.new
+      end
+
+      def locator_params
+        specialty_codes = SPECIALTIES[params[:specialty]]
+        raise Common::Exceptions::InvalidFieldValue.new('specialty', params[:specialty]) if specialty_codes.nil?
+
+        lat, long = coordinates
+        unless lat && long
+          raise Common::Exceptions::UnprocessableEntity.new(
+            detail: 'User has no home latitude and longitude', source: self.class.to_s
+          )
+        end
+
+        {
+          latitude: lat,
+          longitude: long,
+          page: pagination_params[:page_number],
+          per_page: pagination_params[:page_size],
+          radius: RADIUS,
+          specialties: specialty_codes
+        }
+      end
+
+      def coordinates
+        if params[:facilityId]
+          facility = Mobile::FacilitiesHelper.get_facilities(Array(params[:facilityId])).first
```

Comment:
Similar note about `VAOS::V2::MobileFacilityService` since the helper is using the lighthouse API

Reply:
Here we could switch to the new version. However, it led to issues with the spec configuration that I wasn't able to fix easily. There must be some difference in the setup or user. This is an easy change to make later.